eisbuk-admin/cypress/support/index.ts

```diff
@@ -30,3 +30,8 @@ beforeEach(() => {
   );
   cy.stub(reactI18next, "useTranslation").callsFake(() => i18n.t);
 });
+
+afterEach(() => {
+  // auth from previous tests shouldn't affect the following tests
+  cy.signOut();
```

Comment:
No objections to having this, sine explicit is better than implicit, but I'd like to validate my understanding of how Cypress works. Isn't the browser already reset to a clean state between tests, making this not strictly necessary?